File: 200_code_python/19.py
#Viết bằng PYTHON
# Tính số lượng chữ cái trong chuỗi: Nhập một chuỗi và đếm số lượng chữ cái trong chuỗi.

# Nhập chuỗi từ bàn phím
chuoi = input("Nhap mot chuoi: ")

# Đếm số lượng chữ cái trong chuỗi
so_luong_chu_cai = 0
for ky_tu in chuoi:
    if ky_tu.isalpha(): #kiểm tra nếu ký tự là chữ cái
        so_luong_chu_cai += 1 # tăng biến lên 1

# In kết quả
print(f"So luong chu cai trong chuoi la: {so_luong_chu_cai}")

File: 200_code_cpp/19.cpp
// Viết bằng C++
// Tính số lượng chữ cái trong chuỗi: Nhập một chuỗi và đếm số lượng chữ cái trong chuỗi.

#include <iostream>
#include <string>
using namespace std;

int main() {
    // Nhập chuỗi từ bàn phím
    string chuoi;
    cout << "Nhap chuoi: ";
    getline(cin, chuoi); // Sử dụng getline để nhập chuỗi có chứa khoang trắng

    // Đếm số lượng chữ cái trong chuỗi
    int so_luong_chu_cai = 0;
    for (char ky_tu : chuoi) {
        if (isalpha(ky_tu)) { //Kiểm tra nếu ký tự là chữ cái.
            so_luong_chu_cai ++; // Tăng biến đếm lên 1.
        }
    }

    // In kết quả
    cout << "So luong chu cai trong chuoi la: " << so_luong_chu_cai << endl;

    return 0;
}